src/common.rs

```diff
@@ -421,7 +421,11 @@ named!(pub type_identifier<&[u8], SqlType>,
            )
          | do_parse!(
                tag_no_case!("datetime") >>
-               (SqlType::DateTime)
+               len: opt!(delimited!(tag!("("), digit, tag!(")"))) >>
+               (SqlType::DateTime(match len {
+                   Some(len) => len_as_u16(len),
+                   None => 32 as u16,
```

Comment:
It turns out that the "length" of a `DATETIME` is the ["fractional seconds precision" (fsp)](https://dev.mysql.com/doc/refman/8.0/en/date-and-time-type-overview.html), and the default is zero.

Can we change the default to 0, and name the parameter `fsp` or `precision`?